crates/macro-support/src/parser.rs

```diff
@@ -687,17 +687,24 @@ impl<'a> ConvertToAst<(&ast::Program, BindgenAttrs, &'a Option<ast::ImportModule
     }
 }
 
-impl ConvertToAst<(&ast::Program, BindgenAttrs)> for syn::ForeignItemType {
+impl<'a> ConvertToAst<(&ast::Program, BindgenAttrs, &'a Option<ast::ImportModule>)>
+    for syn::ForeignItemType
+{
     type Target = ast::ImportKind;
 
     fn convert(
         self,
-        (program, attrs): (&ast::Program, BindgenAttrs),
+        (program, attrs, module): (&ast::Program, BindgenAttrs, &'a Option<ast::ImportModule>),
     ) -> Result<Self::Target, Diagnostic> {
         let js_name = attrs
             .js_name()
             .map(|s| s.0)
             .map_or_else(|| self.ident.to_string(), |s| s.to_string());
+        let path_name = match module {
+            Some(ast::ImportModule::RawNamed(path, _)) => Some(path.clone()),
+            Some(ast::ImportModule::Named(path, _)) if !path.starts_with("/") => Some(path.clone()),
```

Comment:
```suggestion
            Some(ast::ImportModule::Named(path, _)) if !path.starts_with('/') => Some(path.clone()),
```
I'm not sure why checking for `starts_with('/')` isn't required elsewhere in the code.